torch/distributed/_composable/fsdp/_fsdp_init.py

```diff
@@ -16,3 +19,14 @@ def _normalize_device(device: DeviceLikeType) -> torch.device:
         return torch.device(device)
     else:
         raise TypeError(f"Invalid type for device {device}: {type(device)}")
+
+
+def _init_default_fully_shard_mesh(device_type: str) -> DeviceMesh:
+    """The default fully-shard mesh shards over the global mesh."""
+    default_pg = dist.distributed_c10d._get_default_group()
+    mesh = init_device_mesh(
+        device_type=device_type,
+        mesh_shape=(default_pg.size(),),
+        mesh_dim_names=("dp_shard",),
```

Comment:
wonder if we ever plan to access mesh dim by ``"dp_shard"`` ? curious about the consistency of dim name between user-defined mesh and default mesh

Reply:
Good question. I can remove this mesh dim name for consistency with a user-passed in one. There is no use for it really.